scripts/uniffi_bindgen_generate_kotlin.sh

```diff
@@ -1,15 +1,23 @@
 #!/bin/bash
```

Comment:
the script produces `.dylib` when running on macOS and `.so` on Linux. is there a build flag that we can set to have them always produce `.so`?

Reply:
Actually I didn't realize but yes for us the JVM files on macOS are `.dylib`. The Android files however are `.so` files for all 3 architectures.

Reply:
Oh my bad! I see this is the script to generate the JVM stuff, not the Android library. Looking at the Android shell script all 3 files are also `.so`, so my comment doesn't make sense, because I was looking at the wrong file. All good!